superflore/generators/bitbake/ros_meta.py

```diff
@@ -42,9 +42,6 @@ def clean_ros_recipe_dirs(self, distro=None):
 
```

Comment:
Squash 370fecee7cea1dad03b8f4930a0de1d91d7c78e3 and 0536e6c12fb9963a28489f939bce0687b2fa1222 (but not  5ccff152412edca022ac95fe53abeeab2223ac33).

Reply:
Why? It will all be squashed when the PR is merged, won't be?

Reply:
No. I plan to use the "rebase and merge" option, which doesn't squash.